kong/plugins/rate-limiting/policies/init.lua

```diff
@@ -167,7 +167,12 @@ return {
         current_metric = nil
       end
 
-      return current_metric and current_metric or 0
+      local ok, err = red:set_keepalive(10000, 100)
```

Comment:
There should not be such arbitrary socket timeout and connection pool size arguments unless configurable by the user. Since that is not the case, better not give those and rely on the related ngx_lua directives the user might have configured.